### PR TITLE
WD-10435 - intel iot page rebrand

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Rebrand intel-iot page.

## QA

- [demo link](https://ubuntu-com-13970.demos.haus/download/iot/intel-iot)
- [copy doc](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit)
- [figma](https://www.figma.com/design/cVGimTPHky4h3BPXfrT6GB/24.04-U.com---download-(rebranding)?node-id=1702-5346&m=dev)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correctly updated according to copy doc and figma

## Issue / Card
[WD-10435](https://warthogs.atlassian.net/browse/WD-10435)

[WD-10435]: https://warthogs.atlassian.net/browse/WD-10435?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by juanruitina** [comment]
There were a handful of repetitive small tweaks needed, so I went ahead and applied them myself. Hope that's OK.

There's some weird behaviour of the equal heights component on medium screens (see below), what can we do about it? Just stacking the three boxes would do the trick for me. / @lyubomir-popov 

Screenshot 2024-06-21 at 12 20 39

**Comment by lyubomir-popov** [comment]
this h2 needs to be wrapped in a p-section--shallow:
image

**Comment by lyubomir-popov** [comment]
> There's some weird behaviour of the equal heights component on medium screens (see below), what can we do about it? Just stacking the three boxes would do the trick for me. / @lyubomir-popov

I think they are just stacking, that's why it looks a little under designed. Or did you have something else in mind?

**Comment by lyubomir-popov** [comment]
@juanruitina oh the markup is incorrect, this is what it should be like: 
image

**Comment by lizzochek** [comment]
@juanruitina @lyubomir-popov updated according to your comments

**Comment by lyubomir-popov** [comment]
these two should be in a single code snippet, at the moment theres two `pre`s, causing too much space between lines
image

this looks wrong - 2 "ands", 3 times the amount of text that should be in a title - @Sophie-32 is it possible to shorten or breakoff part of the heading and put it in a paragraph following the heading?:
image

the last section before the footer on every page should have p-section--deep on it, to doulbe the amount of white space at the bottom:
image

File: templates/download/iot/intel-iot.html [hunk 1]
```diff
@@ -1,426 +1,1071 @@
...
+                  <li class="p-inline-list__item">
+                    <a aria-label="Instructions for Ubuntu Server"
+                       href="https://assets.ubuntu.com/v1/a0b71e1e-Installation+Instructions+Server+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Docs for Ubuntu Server" href="/server/docs">Docs&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:485]: please remove the white space here

File: templates/download/iot/intel-iot.html [hunk 2]
```diff
@@ -1,426 +1,1071 @@
...
+                       href="https://assets.ubuntu.com/v1/2715b851-Installation+Instructions+Desktop+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Learn more about Ubuntu Desktop"
+                       href="/desktop/developers">Learn
+                    more&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://discourse.ubuntu.com/t/ubuntu-24-04-lts-noble-numbat-release-notes/39890">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:285]: please remove a white space here

File: templates/download/iot/intel-iot.html [hunk 3]
```diff
@@ -1,426 +1,1071 @@
...
+      </div>
+      <div class="col">
+        <p>
+          Optimised <a href="/real-time">Real-time Ubuntu</a> is now production-ready on:
+        </p>
+        <ul class="p-list">
+          <li class="p-list__item has-bullet">Intel Atom&reg; X6000E Series Processors (codename Elkhart Lake)</li>
+          <li class="p-list__item has-bullet">11th Gen Intel&reg; Core&trade; processors (codename Tiger Lake)</li>
+          <li class="p-list__item has-bullet">
+            12th Gen Intel&reg; Core&trade; processors (code name Alder Lake S and Alder Lake N)
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:97]:
```suggestion
            12th Gen Intel&reg; Core&trade; processors (codename Alder Lake S and Alder Lake N)
```

File: templates/download/iot/intel-iot.html [hunk 4]
```diff
@@ -1,426 +1,1071 @@
...
+      </div>
+      <div class="col">
+        <p>
+          11th Gen Intel&reg; Core&trade; processors (formerly known as Tiger Lake &mdash; TGL) deliver a balance of
+          performance and responsiveness in a low-power platform built on our third-generation 10nm process technology.
+          Engineered to deliver for IoT markets, these processors can support low-latency and time-sensitive applications,
+          and have the power to run multiple workloads including AI and deep learning applications on a single platform.
+        </p>
+        <hr class="is-muted" />
+        <a aria-href="https://www.intel.com/content/www/us/en/products/docs/processors/core/11th-gen-core-enhanced-for-iot-brief.html">Read
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:1014]:
```suggestion
        <a href="https://www.intel.com/content/www/us/en/products/docs/processors/core/11th-gen-core-enhanced-for-iot-brief.html">Read
```

**Review by akbarkz — CHANGES_REQUESTED** [review]

File: templates/download/iot/intel-iot.html [hunk 5]
```diff
@@ -1,426 +1,1067 @@
...
+                       href="https://assets.ubuntu.com/v1/2715b851-Installation+Instructions+Desktop+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Learn more abbout Ubuntu Desktop"
+                       href="/desktop/developers">Learn
+                    more&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:454]: white space

File: templates/download/iot/intel-iot.html [hunk 6]
```diff
@@ -1,426 +1,1067 @@
...
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:382]: white space

File: templates/download/iot/intel-iot.html [hunk 7]
```diff
@@ -1,426 +1,1067 @@
...
+                  <li class="p-inline-list__item">
+                    <a aria-label="Instructions for Ubuntu Server"
+                       href="https://assets.ubuntu.com/v1/a0b71e1e-Installation+Instructions+Server+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Docs for Ubuntu Server" href="/server/docs">Docs&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://discourse.ubuntu.com/t/ubuntu-24-04-lts-noble-numbat-release-notes/39890">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:312]: white space

File: templates/download/iot/intel-iot.html [hunk 8]
```diff
@@ -1,426 +1,1067 @@
...
+                    <a aria-label="Docs for Ubuntu Server" href="/server/docs">Docs&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
+                    22.04 LTS&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/fd2fa325-LookoutCanyon_ReleaseNotes_20.04_221115_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:902]: white space

File: templates/download/iot/intel-iot.html [hunk 9]
```diff
@@ -1,426 +1,1067 @@
...
+                  <li class="p-inline-list__item">
+                    <a aria-label="Instructions for Ubuntu Server"
+                       href="https://assets.ubuntu.com/v1/a0b71e1e-Installation+Instructions+Server+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Docs for Ubuntu Server" href="/server/docs">Docs&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:897]: white space

File: templates/download/iot/intel-iot.html [hunk 10]
```diff
@@ -1,426 +1,1067 @@
...
+                    more&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
+                    22.04 LTS&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/fd2fa325-LookoutCanyon_ReleaseNotes_20.04_221115_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:864]: white space

File: templates/download/iot/intel-iot.html [hunk 11]
```diff
@@ -1,426 +1,1067 @@
...
+                       href="https://assets.ubuntu.com/v1/2715b851-Installation+Instructions+Desktop+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Learn more about Ubuntu Desktop"
+                       href="/desktop/developers">Learn
+                    more&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:859]: white space

File: templates/download/iot/intel-iot.html [hunk 12]
```diff
@@ -1,426 +1,1067 @@
...
+                    <a aria-label="Docs for Ubuntu Server" href="/server/docs">Docs&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
+                    22.04 LTS&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/fd2fa325-LookoutCanyon_ReleaseNotes_20.04_221115_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:778]: white space

File: templates/download/iot/intel-iot.html [hunk 13]
```diff
@@ -1,426 +1,1067 @@
...
+                    more&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
+                    22.04 LTS&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/fd2fa325-LookoutCanyon_ReleaseNotes_20.04_221115_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:740]: white space

File: templates/download/iot/intel-iot.html [hunk 14]
```diff
@@ -1,426 +1,1067 @@
...
+                       href="https://assets.ubuntu.com/v1/2715b851-Installation+Instructions+Desktop+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Learn more about Ubuntu Desktop"
+                       href="/desktop/developers">Learn
+                    more&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:735]: white space

File: templates/download/iot/intel-iot.html [hunk 15]
```diff
@@ -1,426 +1,1067 @@
...
+                    <a aria-label="Instructions for Ubuntu Server"
+                       href="https://assets.ubuntu.com/v1/a0b71e1e-Installation+Instructions+Server+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Docs for Ubuntu Server" href="/server/docs">Docs&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes
+                      for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:654]: white space

File: templates/download/iot/intel-iot.html [hunk 16]
```diff
@@ -1,426 +1,1067 @@
...
+                       href="https://assets.ubuntu.com/v1/2715b851-Installation+Instructions+Desktop+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Learn more abbout Ubuntu Desktop"
+                       href="/desktop/developers">Learn
+                    more&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:624]: white space

File: templates/download/iot/intel-iot.html [hunk 17]
```diff
@@ -1,426 +1,1067 @@
...
+                    <a aria-label="Instructions for Ubuntu Server"
+                       href="https://assets.ubuntu.com/v1/a0b71e1e-Installation+Instructions+Server+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Docs for Ubuntu Server" href="/server/docs">Docs&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes
+                      for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:581]: white space

File: templates/download/iot/intel-iot.html [hunk 18]
```diff
@@ -1,426 +1,1067 @@
...
+                       href="https://assets.ubuntu.com/v1/2715b851-Installation+Instructions+Desktop+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Learn more abbout Ubuntu Desktop"
+                       href="/desktop/developers">Learn
+                    more&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:551]: white space

File: templates/download/iot/intel-iot.html [hunk 19]
```diff
@@ -1,426 +1,1067 @@
...
+                  <li class="p-inline-list__item">
+                    <a aria-label="Instructions for Ubuntu Server"
+                       href="https://assets.ubuntu.com/v1/a0b71e1e-Installation+Instructions+Server+Image+Intel+IoT+Platforms.pdf">Instructions&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a aria-label="Docs for Ubuntu Server" href="/server/docs">Docs&nbsp;&rsaquo;</a>
+                  </li>
+                  <li class="p-inline-list__item">
+                    <a href="https://assets.ubuntu.com/v1/66ac0a18-LookoutCanyon_ReleaseNotes_22.04_230330_V1.1.pdf">Release
+                      notes for
```
Comment (akbarkz) [inline templates/download/iot/intel-iot.html:411]: white space

**Review by akbarkz — APPROVED** [review]

**Pull request merged**